common/lib/models/comment.dart

```diff
@@ -39,6 +42,7 @@ class Comment extends CommentBase {
   const Comment({
     required this.id,
     required this.pollId,
+    required this.parentId,
```

Comment:
Since this field is nullable, don't mark it with `required`.

Reply:
Done! I did this in multiple places, so will go back and try to catch 'em all.